content/en/docs/ops/best-practices/security/index.md

```diff
@@ -10,6 +10,14 @@ test: no
 Istio security features provide strong identity, powerful policy, transparent TLS encryption, and authentication, authorization and audit (AAA) tools to protect your services and data.
 However, to fully make use of these features securely, care must be taken to follow best practices. It is recommended to review the [Security overview](/docs/concepts/security/) before proceeding.
 
+## Envoy Downstream Connections
```

Comment:
@jacob-delgado From a conversation offline with @craigbox:

* Can we move this further down the page? 
* Can we rename it to "Configure a limit on downstream connections"
* Can we add some context on why users want to limit the number of connections they support?